backend/plugins/content_api/src/modules/knowledgebase/graphql/resolvers/queries.ts

```
@@ -0,0 +1,266 @@
+  const qry: any = {};
+
+  const keys = ['codes', 'categoryIds', 'articleIds', 'topicIds'];
+
+  keys.forEach((key) => {
+    if (args[key] && args[key].length > 0) {
+      const field = key.replace('s', '');
+      qry[field] = { $in: args[key] };
+    }
+  });
+
+  if (args.searchValue && args.searchValue.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)